RadarSDK/RadarAPIClient.m

```diff
@@ -177,13 +192,15 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
         tripOptionsDict[@"mode"] = [Radar stringForMode:tripOptions.mode];
         params[@"tripOptions"] = tripOptionsDict;
     }
+    
     RadarTrackingOptions *options = [RadarSettings trackingOptions];
     if (options.syncGeofences) {
         params[@"nearbyGeofences"] = @(YES);
     }
     if (nearbyBeacons) {
         params[@"nearbyBeacons"] = nearbyBeacons;
     }
+    params[@"trackingOptions"] = [options dictionaryValue];
```

Comment:
This is sent up and serialized to our locations cold storage for debugging.

Reply:
add as comment?